src/main/webapp/js/instructorFeedbacks.js

```diff
@@ -88,12 +88,26 @@ function selectDefaultTimeOptions() {
         $('#' + FEEDBACK_SESSION_TIMEZONE).val(timeZone);
     }
 
+    if (isUtcDateDifferentFromCurrentDate(currentTime, timeZone)) {
+        shiftDateToToday();
+    }
+
     var uninitializedTimeZone = $('#timezone > option[value=\'' + TIMEZONE_SELECT_UNINITIALISED + '\']');
     if (uninitializedTimeZone) {
         uninitializedTimeZone.remove();
     }
 }
 
+function isUtcDateDifferentFromCurrentDate(time, timeZone) {
+    var isUtcAhead = (timeZone - time) <= -24;
+    var isUtcBehind = (timeZone - time) >= 1; // this is not zero because we are using next nearest hour
+    return isUtcAhead || isUtcBehind;
+}
+
+function shiftDateToToday() {
+    var currentDate = new Date(new Date().getTime());
+    $('#startdate').val(convertDateToDDMMYYYY(currentDate));
+}
```

Comment:
It would be nice if the logic can be separated out into a method that can be tested independently using QUnit tests.